crates/red_knot_python_semantic/src/types/class.rs

```diff
@@ -43,6 +43,14 @@ pub struct Class<'db> {
     pub(crate) known: Option<KnownClass>,
 }
 
+impl<'db> Class<'db> {
```

Comment:
Why a new separate `impl` block for `Class`, rather than adding this to the existing impl block below?

Reply:
Because Micha blind